### ISSUE TITLE
New Rule T0019: Indent operator correcty

### ISSUE THREAD

**Issue by pavel-mikula-sonarsource** [SonarSource/sonar-dotnet#9152]
Coding style:

> Indented at the same level if the syntax at the beginning of the previous line is a sibling.
```
void Foo() =>
    A
    && B; // A and B are siblings => we don't indent
```
Indented one level further otherwise.
```
return A
    && B; // "return" is the parent of A and B => we indent
```

Messages:
Indent operator at the same position as the previous line
Indent operator +4 further than the previous line

This rule does not care about the position of &&, that is task for T0018. The rule should trigger only if the operator is the first token on the line.

**Comment by pavel-mikula-sonarsource** [comment]
https://sonarsource.atlassian.net/browse/NET-1329
